dashboard/db/migrate/20160617163547_create_user_geos.rb

```diff
@@ -0,0 +1,16 @@
+class CreateUserGeos < ActiveRecord::Migration
+  def change
+    create_table :user_geos do |t|
+      t.references :user, index: true, unique: true, foreign_key: true, null: false
+      t.timestamps null: false
+      t.datetime :indexed_at, null: false
+      t.string :ip_address
+      t.string :city
+      t.string :state
+      t.string :country
+      t.string :postal_code
+      t.decimal :latitude, precision: 8, scale: 6
+      t.decimal :longitude, precision: 9, scale: 6
```

Comment:
Both give six digits after the decimal point, since latitude is between -90 and 90 and longitude is between -180 and 180.